pax-runtime/src/engine/mod.rs

```diff
@@ -367,14 +368,16 @@ impl PaxEngine {
             // OBS: HACK: this is not general, works for non-for loop/if nodes only
             // to do more generally, split expanded_node.update into prop updates and
             // regen of children steps
-            Rc::clone(&parent).recurse_unmount(ctx);
-            let env = Rc::clone(&parent.stack);
-            let parent_template = borrow!(parent.instance_node);
-            let children = borrow!(parent_template.base().get_instance_children());
-            let new_templates = children.clone().into_iter().zip(iter::repeat(env));
-            let children = parent.generate_children(new_templates, ctx);
-            parent.children.set(children);
-            Rc::clone(&parent).recurse_mount(ctx);
+            // Rc::clone(&parent).recurse_unmount(ctx);
```

Comment:
another case of commenting out vs getting rid of. Unless you plan on reverting this soon, let's remove comments like this please.

Reply:
Done!